SU-SWS/stanford_basic
# D8CORE-1402: Fix for IE footer

# READY FOR REVIEW

# Summary
- Fix allowing you to be able to read footer items in IE11

# Review By (Date)
- Before code freeze

# Urgency
- Not

# Steps to Test

1. Look at footer in IE11
1. Check out this branch
1. Rebuild Cache
1. Look at the footer in IE11 again
1. Verify that you can read footer items.

![Screen Shot 2020-04-09 at 4 58 01 PM](https://user-images.githubusercontent.com/284440/78950349-598def80-7a83-11ea-8f5f-fdc44fcfb539.png)



# Affected Projects or Products
- D8Core, Maybe Decanter?

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1402
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)

This is a bit of a hack because it doesn't put footer items in the correct order. However because this is for IE11, I didn't know if I should dedicate any more time to this.


- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
